examples/generate_config.sh

```diff
@@ -104,9 +104,6 @@ zero="\
   if [ $ZERO_STAGE == 1 ]; then
     if [ $PP > 1 ]; then
     extra="\
-        \"data_types\": {
-          \"grad_accum_dtype\": \"fp32\"
```

Comment:
By removing the flags, it means the grad_accm_dtype will take the specified dtype in model which is bf16 in our case?

Reply:
Yes, the gard_accm_dtype will align with your dataype, which is bf16 in our case.